src/eval.rs

```diff
@@ -1571,14 +1571,13 @@ fn apply_continuation<F: LurkField>(
                             let scalar_ptr = store
                                 .get_expr_hash(result)
                                 .ok_or_else(|| LurkError::Store("expr hash missing".into()))?;
-                            store.get_char(
-                                char::from_u32(
-                                    scalar_ptr.value().to_u32().ok_or_else(|| {
-                                        LurkError::Eval("Ptr is invalid u32".into())
-                                    })?,
+
+                            let res = scalar_ptr.value().to_u32_unchecked();
+                            store.get_char(char::from_u32(res).ok_or_else(|| {
+                                LurkError::Store(
+                                    "LurkField::to_u32_unchecked invariant violated".into(),
```

Comment:
I think failure here actually means the U32 doesn't represent a valid unicode character. #231 updates this behavior so failure is impossible, which is the desired behavior. (We want all u32 to have corresponding chars.)

Reply:
I changed the message and inserted a comment to help while merging #231.